.teamcity/src/main/kotlin/projects/GradleBuildToolRootProject.kt

```diff
@@ -20,8 +20,8 @@ class GradleBuildToolRootProject(branch: VersionedSettingsBranch) : Project({
     val gradleBuildBucketProvider = StatisticBasedFunctionalTestBucketProvider(model, File("./test-class-data.json"))
     subProject(CheckProject(model, gradleBuildBucketProvider))
 
-    if (model.branch != VersionedSettingsBranch.EXPERIMENTAL) {
-        subProject(PromotionProject(model.branch))    
+    if (!model.branch.isExperimental) {
```

Comment:
I think we still need promotion build configurations if we want to test the promotion changes, i.e. what we're currently do - remove `buildReceipt`.